collections/_release_4_5/entry-systems-xr-fragment-density-map-support.md

```diff
@@ -0,0 +1,20 @@
+---
+type: entry
+section: systems
+subsection: xr
+rank: 0
+importance: 4
+anchor: fragment-density-map-support
+title: Fragment density map support
+blockquote: TODO
+text: |
+  In order to push great visuals on a VR headset, it is OK to cheat a little. As players tend to turn their head rather than move their eyes too far from the center and as pixels around the outside of the viewport are often distorted by the lens, why should the device render these parts at full resolution?
```

Comment:
```suggestion
  In order to push great visuals on a VR headset, a little cheating is often necessary. As players tend to turn their head rather than move their eyes too far from the center and as pixels around the outside of the viewport are often distorted by the lens, why should the device render these parts at full resolution?
```